### ISSUE TITLE
ETIMEDOUT err.code causes app to fail

### ISSUE THREAD

**Issue by jchblsc** [watson-developer-cloud/assistant-simple#232]
In the following in `app.js:`

```
assistant.message(payload, function (err, data) {
    if (err) {
      return res.status(err.code || 500).json(err);
    }

    return res.json(data);
});
```

The returned `err.code` is not always a valid HTTP return code.  

For example, ETIMEDOUT error has an associated `err.code` of 0 and causes the entire Node app to go belly up:  

```
_http_server.js:195
    throw new RangeError(`Invalid status code: ${statusCode}`);
    ^

RangeError: Invalid status code: 0
    at ServerResponse.writeHead (_http_server.js:195:11)
    at ServerResponse._implicitHeader (_http_server.js:159:8)
    at ServerResponse.end (_http_outgoing.js:621:10)
    at ServerResponse.send (D:\jch\Prog\Node\eclipse\workspace\Jch-k8s-banking-assistant\node_modules\express\lib\response.js:221:10)
```

**Comment by Oliver-I** [comment]
@stevenpkg Can you take a look at this?

**Comment by Oliver-I** [comment]
@jchblsc This has now been resolved with pull request #233
